io.github.hermitdemschoenenleben.linien.yml

```
@@ -0,0 +1,112 @@
+app-id: io.github.hermitdemschoenenleben.linien
+runtime: org.kde.Platform
+runtime-version: '5.12'
```

Update runtime to 5.14
```suggestion
runtime-version: '5.14'
```